src/components/AdminSettings/SIPBridge.vue

```diff
@@ -0,0 +1,170 @@
+<!--
+ - @copyright Copyright (c) 2019 Joas Schilling <coding@schilljs.com>
```

Comment:
2020 is almost over :wink:

Reply:
Maybe I started a year ago already? :P

Reply:
https://github.com/nextcloud/spreed/pull/4496/commits/517a79cc481038e27b6ddedb32301c7691433627